zubhub_frontend/zubhub/src/views/edit_profile/EditProfile.jsx

```diff
@@ -363,63 +368,186 @@ function EditProfile(props) {
                     </FormControl>
                   </Grid>
 
-                  <Grid item xs={12} sm={6} md={6}>
-                    <FormControl
-                      className={clsx(classes.margin, classes.textField)}
-                      variant="outlined"
-                      size="small"
-                      fullWidth
-                      margin="normal"
-                      error={
-                        (props.status && props.status['password']) ||
-                        (props.touched['password'] && props.errors['password'])
-                      }
-                    >
-                      <InputLabel
-                        className={classes.customLabelStyle}
-                        htmlFor="password"
+                  <Grid container style={{ display: 'block'}}>            
```

Comment:
Why are you changing the grid system to block? its just gonno affect responsiveness

FYI If you aare changing to block system just use div instead